.github/workflows/ci_linux.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       run: |
         sudo apt -y update
         sudo apt -y install meson ninja-build
-        sudo apt -y install libsdl2-dev
+        sudo apt -y install libsdl2-dev libva-dev
```

Comment:
You probably want to add `-Dvaapi=enabled` to the build to avoid that kind of issue in the future.

Reply:
Done